tests/Cluster.py

```diff
@@ -46,11 +46,18 @@ class Cluster(object):
     __BiosPort=8788
     __LauncherCmdArr=[]
     __bootlog="eosio-ignition-wd/bootlog.txt"
```

Comment:
Wouldn't this now be "eosio-ignition-wd/cluster00000/bootlog.txt" for cluster 0? Also is this even needed? Shouldn't it be replaced during initialization?